src/follower.py

```diff
@@ -44,6 +45,10 @@ def __init__(self):
         self._turn_angle = 0
         self._decision = 0
 
+        self._turn_right = 0
```

Comment:
Delete these three variables. You're not using them.

Reply:
deleted